pkg/services/k8s/resources/librarypanel/watcher.go

```diff
@@ -1,26 +1,44 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
+//
```

Comment:
> this watcher implementation is generated once if it doesn't exist, and it is meant to be edited.

Yeah, codejen is intentionally designed to _not_ support this case, as i believe it's an antipattern; some more discussion in [this thread](https://github.com/grafana/grafana/pull/54816#issuecomment-1255953200).

To provide the best experience for people actually writing these, my preference would be to avoid generating the stub, and instead split the problem here into:
* The Go API surface (funcs, types) that needs to be presented from for each kind. These would be generated by the jenny for each kind in the appropriate dir
* Parts that absolutely must be implemented by the kind author. Control is passed from the generated Go API surface funcs to something the user can edit by way of a generated call to an unexported func.
* Helpers that can generically cover either part of the implementation, or provide a complete default implementation that the dev working on that kind can then optionally disable 

#59899 illustrated this approach.

Not following the basic pattern would have DX implications for `make gen-cue` and similar. But, i also don't want to dictate implementation, so i mostly offer the above as a step towards looking for consensus on approach